website/templates/docs.html

```diff
@@ -535,12 +539,19 @@ <h2 id=generics>Generics</h2>
 }
 
 <comment>// Generic code is notoriously verbose. To reduce clutter, V doesn't require you 
-// to add `&lt;T&gt;` every time, since it already knows that Repo is a generic type.</comment> 
+// to add `&lt;T&gt;` every time, since it already knows that Repo is a generic type.
+// Here the function return type is an unspecified instance of Repo.</comment> 
 fn new_repo&lt;T>(db DB) Repo {
 	return Repo&lt;T>{db: db}
 }
 
-<comment>// This is a generic function. V will generate it for every type it's used with.</comment> 
+fn (db DB) query_one&lt;T>(query string, id int) T? {
+	<comment>// Return a value of type T from the database, or an error.
+	// Implementation omitted...</comment>
+}
+
+<comment>// This is a generic receiver function. V will generate it for every Repo instance it's used with.
+// T is a member type of Repo, so it is in scope for this receiver function.</comment> 
```

Comment:
This is my understanding anyway ;-)
This code suprised me a bit. I would have thought that only members of `r` would be in scope for this function, not members of `Repo`. Or is `T` a member of `r` too?

Reply:
T is not a member, it's a type. I'll make it more clear.

Reply:
I assumed T was exposed as `Repo.T`, but maybe V just makes it accessible for this function because Repo was mentioned as the receiver parameter and T is the template parameter for Repo?

Reply:
I think I understand now, see #13.